lldb/include/lldb/Interpreter/CommandInterpreter.h

```diff
@@ -766,6 +771,14 @@ class CommandInterpreter : public Broadcaster,
   CommandUsageMap m_command_usages;
 
   StreamString m_transcript_stream;
```

Comment:
We should mention in a comment that this will only be filled in if the setting is true.